homeassistant/components/homekit_controller/cover.py

```diff
@@ -214,34 +214,37 @@ def is_vertical_tilt(self) -> bool:
     @property
     def current_cover_tilt_position(self) -> int | None:
         """Return current position of cover tilt."""
-        tilt_position = self.service.value(CharacteristicsTypes.VERTICAL_TILT_CURRENT)
-        if not tilt_position:
+        tilt_position = None
+        minValue = None
```

Comment:
Can you use min_value and max_value here.

Reply:
Thanks for the review. 
Added a test case if min/max is not set = default values
Fixed case